src/commonMain/kotlin/exchange.dydx.abacus/output/Compliance.kt

```diff
@@ -19,6 +19,21 @@ enum class ComplianceStatus(val rawValue: String?) {
     }
 }
 
+@JsExport
+@Serializable
+enum class ComplianceAction(val rawValue: String?) {
+    ONBOARD("ONBOARD"),
+    CONNECT("CONNECT"),
+    VALID_SURVEY("VALID_SURVEY"),
+    INVALID_SURVEY("INVALID_SURVEY"),
+    ;
+
+    companion object {
+        operator fun invoke(rawValue: String?) =
+            ComplianceStatus.values().firstOrNull { it.rawValue == rawValue }
+    }
```

Comment:
kotlin tip: Don't actually need to have a `rawValue` and a custom method to parse. Kotlin will automatically convert the enum type <> String:
https://kotlinlang.org/docs/enum-classes.html#working-with-enum-constants

Basically you can remove the string values from the enum's constructor, then use `ComplianceAction.valueOf(rawValue)`

Reply:
do you mean just like:
`enum ComplianceAction { ONBOARD, CONNECT, ... }`

I am following the pattern for enums at other places I wonder if there is a reason we always use `rawValue`